libs/vm_modules/examples/commodity_prediction/script/commodity_prediction.etch

```diff
@@ -0,0 +1,64 @@
+function main()
+
+if (System.Argc() != 2)
+  print("Usage: VM SCRIPT_FILE PATH/TO/model_files/ /PATH/TO/INPUT/file.csv");
+  return;
+endif
+
+var loader = CommodityDataLoader();
+loader.AddData(System.Argv(1));
+
+//print(loader.GetNext().ToString());
+
+// mkae graph with correct architecture
+var graph = Graph();
+
+graph.AddPlaceholder("num_input");
+graph.AddDropout("dropout_0", "num_input", 1.0f);
+graph.AddFullyConnected("hidden_dense_1", "dropout_0", 118, 216);
```

Comment:
setup variables ahead of time so that these hardcoded values don't seem arbitrary. e.g.:
var n_features = 118;
var hidden_layer_1_size = 216;
etc.